helpers/helper-translation-discussion.php

```diff
@@ -214,6 +214,23 @@ public function sanitize_comment_topic( $comment_topic ) {
 		return $comment_topic;
 
 	}
+
+	public function sanitize_comment_locale( $comment_locale ) {
+		$gp_locales     = new GP_Locales();
+		$all_gp_locales = array_keys( $gp_locales->locales );
+
+		if ( ! in_array( $comment_locale, $all_gp_locales ) ) {
+			$comment_locale = 'unknown';
+		}
+		return $comment_locale;
+	}
+
+	public function sanitize_translation_id( $translation_id ) {
+		if ( ! is_numeric( $translation_id ) ) {
```

Comment:
Check for a valid translation like this?
```suggestion
		if ( $translation_id! GP::$translation->get( $translation_id ) ) {
```